irohad/model/converters/impl/pb_command_factory.cpp

```diff
@@ -19,12 +19,55 @@
 #include "model/converters/pb_common.hpp"
 
 #include <commands.pb.h>
-#include <string>
+#include <model/permissions.hpp>
+#include <set>
+
+#include <boost/assign/list_inserter.hpp>
 
 namespace iroha {
   namespace model {
     namespace converters {
 
+      PbCommandFactory::PbCommandFactory() {
+
+        boost::assign::insert(pb_role_map_)
+            (protocol::RolePermission::can_get_my_acc_ast, can_get_my_acc_ast)
+            (protocol::RolePermission::can_get_my_signatories, can_get_my_signatories)
+            (protocol::RolePermission::can_set_quorum, can_set_quorum)
+            (protocol::RolePermission::can_get_my_acc_txs, can_get_my_acc_txs)
+            (protocol::RolePermission::can_get_my_account, can_get_my_account)
+            (protocol::RolePermission::can_get_all_acc_ast, can_get_all_acc_ast)
+            (protocol::RolePermission::can_get_all_acc_ast_txs, can_get_all_acc_txs)
+            (protocol::RolePermission::can_get_all_acc_txs, can_get_all_acc_txs)
+            (protocol::RolePermission::can_get_all_accounts, can_get_all_accounts)
+            (protocol::RolePermission::can_remove_signatory, can_remove_signatory)
+            (protocol::RolePermission::can_add_signatory, can_add_signatory)
+            (protocol::RolePermission::can_create_domain, can_create_domain)
+            (protocol::RolePermission::can_create_account, can_create_account)
+            (protocol::RolePermission::can_set_quorum, can_set_quorum)
+            (protocol::RolePermission::can_add_peer, can_add_peer)
+            (protocol::RolePermission::can_add_asset_qty, can_add_asset_qty)
+            (protocol::RolePermission::can_append_role, can_append_role)
+            (protocol::RolePermission::can_create_asset, can_create_asset)
+            (protocol::RolePermission::can_create_role, can_create_role)
+            (protocol::RolePermission::can_get_all_signatories, can_get_all_signatories)
+            (protocol::RolePermission::can_get_my_acc_ast_txs, can_get_my_acc_ast_txs)
+            (protocol::RolePermission::can_transfer, can_transfer)
+            (protocol::RolePermission::can_receive, can_receive)
+            (protocol::RolePermission::can_read_assets, can_read_assets)
+            (protocol::RolePermission::can_grant_set_quorum, "CanGrant"+can_set_quorum)
```

Comment:
maybe short explanation is needed about why we put CanGrant in front to distinct grantable rights